app/routes/_index/route.tsx

```diff
@@ -1,3 +1,22 @@
+import { CategoryLink } from '~/components/category-link/category-link';
+import styles from './index.module.scss';
+import heroImageJpg from '~/assets/img/heroImage.jpg';
+import { LabelWithArrow } from '~/components/label-with-arrow/label-with-arrow';
+
 export default function HomePage() {
-    return <h1>Home</h1>;
+    return (
+        <>
+            <div className={styles.heroImage}>
+                <img src={heroImageJpg} alt="Hero banner" />
```

Comment:
That's such a useless `alt`, it's worse than no alt at all. The alt attribute is for images that are not purely decorative. It's for screen readers and to display if the image has failed to load. Do you really want screen readers to read "hero banner"? For what?

Reply:
I imagine some blind person who listen about web page content.
To be honest, I've never seen any video about people who need it. 
Will remove.

Reply:
> I imagine some blind person who listen about web page content.

And they need to hear "hero image"?